core/src/main/java/org/apache/iceberg/deletes/BitmapPositionDeleteIndex.java

```diff
@@ -47,4 +47,8 @@ public boolean isDeleted(long position) {
   public boolean isEmpty() {
     return roaring64Bitmap.isEmpty();
   }
+
+  public long numberOfPositionsDeleted() {
```

Comment:
I think this may be ok to include in the interface itself, I can't think of an implementation which couldn't track this?

Reply:
Ok, I can put `numberOfPositionsDeleted()` in the `PositionDeleteIndex` interface.

Reply:
Hmm, turns out there is a `TestSparkParquetReadMetadataColumns.CustomizedPositionDeleteIndex` that implements `PositionDeleteIndex`. I'll leave the interface alone then, since I don't want to change unrelated tests.

Reply:
It makes sense to put it in the interface. The change of `CustomizedPositionDeleteIndex` is minor. 
Minor suggestion: `numberOfPositionsDeleted` -> `numberOfPositionDeletes`

Reply:
Looking a bit more, we don't need this due to https://github.com/apache/iceberg/pull/4588/files#r905365603.